reV/handlers/cli_multi_year.py

```diff
@@ -80,7 +80,7 @@ def from_config(ctx, config_file, verbose):
 
     # initialize loggers.
     init_mult(name, config.logdir,
-              modules=[__name__, 'reV.handlers.multi_year'],
+              modules=[__name__, 'reV'],
```

Comment:
I forgot about the handler CLIs, good catch!

Reply:
I only fixed this here, Let me also fix it in the collection cli...